sdk/python/setup.cfg

```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = oomstore
+version = 0.0.1
+author = oom
+author_email = author@example.com
+description = oomstore python sdk
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/oom-ai/oomstore
+project_urls =
+    Bug Tracker = https://github.com/oom-ai/oomstore/issues
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = oomstore
+packages = find: oomstore
+python_requires = >=3.6
```

Comment:
lets require >= 3.7

3.6 reaches end of life soon https://endoflife.date/python

Reply:
done